Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/Concepts.java

```diff
@@ -189,10 +222,21 @@ private JsonValidationResult checkJsonObject(JSONObject jsonObject) {
                     false);
         }
 
-        if (jsonObject.get(JsonFields.DESCRIPTION) == null) {
+        String pos = jsonObject.get(JsonFields.POS).toString();
+        if (!POS.posValues.contains(pos)) {
+            logger.error("Error creating concept from REST call. " + pos + " does not exist.");
+            return new JsonValidationResult("POS '" + pos + "' does not exist.", jsonObject, false);
```

Comment:
no need to log here, we don't log the other ones. Also, it's definitely not an error that needs to be logged, if anything than maybe debug. 

For the error message, please follow the same format that the other validation messages use.